### PR TITLE
Return data uri image as icon support

### PR DESCRIPTION
**Pull request by NoPlagiarism**
Also, bit of silly sanitizing data (accept only `data:image`)

Any documentation changes needed?

### CODE REVIEW

File: Flow.Launcher.Plugin/Result.cs [hunk 1]
```diff
@@ -70,7 +70,8 @@ public string IcoPath
                     && !string.IsNullOrEmpty(PluginDirectory)
                     && !Path.IsPathRooted(value)
                     && !value.StartsWith("http://", StringComparison.OrdinalIgnoreCase)
-                    && !value.StartsWith("https://", StringComparison.OrdinalIgnoreCase))
+                    && !value.StartsWith("https://", StringComparison.OrdinalIgnoreCase)
+                    && !value.StartsWith("data:image", StringComparison.OrdinalIgnoreCase))
```
[the remark this answers is not in the document]
Comment (NoPlagiarism) [reply Flow.Launcher.Plugin/Result.cs:74]: Uhm.. I don't think this is 100% important. I'll wait for others' opinions

[the remark this answers is not in the document]
Comment (VictoriousRaptor) [reply Flow.Launcher.Plugin/Result.cs:74]: what if it receives an incorrect base64 uri?

[the remark this answers is not in the document]
Comment (NoPlagiarism) [reply Flow.Launcher.Plugin/Result.cs:74]: > what if it receives an incorrect base64 uri?

It goes to fallback


https://github.com/user-attachments/assets/9b473af8-bb6f-4ec9-bb9e-c483c515e418

[the remark this answers is not in the document]
Comment (NoPlagiarism) [reply Flow.Launcher.Plugin/Result.cs:74]: So, I guess it resolved

**Comment by taooceros** [comment]
Don't think we need to change doc. Nobody is using this I think.

**Comment by NoPlagiarism** [comment]
> Nobody is using this I think.

Kinda sad. Kinda truth

**Review by VictoriousRaptor — APPROVED** [review]

**Comment by Garulf** [comment]
This use to work before we added http support and async. Glad to see it back. Thank you.

**Pull request merged**